sorc/fv3nc2nemsio.fd/makefile.sh

```diff
@@ -1,24 +0,0 @@
-#!/bin/ksh
-set -x
```

Comment:
Why is this being removed? Did this code also get removed and/or moved elsewhere?

Reply:
We build it w/ CMake.